ee/indexeddb/values.go

```diff
@@ -26,6 +26,7 @@ const (
 	// numbers
 	tokenInt32  byte = 0x49 // I
 	tokenUint32 byte = 0x55 // U
+	tokenDouble byte = 0x4e // N
```

Comment:
The source for this value, as well as the troubleshooting steps I took to fix this parsing error, are documented here: https://github.com/kolide/launcher/tree/main/ee/katc/test_data#troubleshooting-parsing-errors